README.md

```diff
@@ -29,3 +29,12 @@ Pre-requisites:
 - NetBackup 8.1.1 or higher
 - See script README for perl requirements and usage
 
+#### Executing the snippets using curl
+Pre-requisites:
+- NetBackup 8.1.1 or higher
+- curl 7.51.0 or higher
+- jq command-line parser
```

Comment:
Minor: Can include a link to jq
https://github.com/stedolan/jq/releases

Reply:
added